public/components/FeatureAnywhereContextMenu/AssociatedDetectors/components/__tests__/__snapshots__/ConfirmUnlinkDetectorModal.test.tsx.snap

```diff
@@ -0,0 +1,22 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CodeModal spec renders the component 1`] = `
+<div
+  aria-hidden="true"
+  data-aria-hidden="true"
+/>
+`;
+
+exports[`ConfirmUnlinkDetectorModal spec renders the component 1`] = `
+<div
+  aria-hidden="true"
+  data-aria-hidden="true"
+/>
+`;
+
+exports[`ConfirmUnlinkDetectorModal spec renders the component correctly 1`] = `
+<div
+  aria-hidden="true"
+  data-aria-hidden="true"
+/>
+`;
```

Comment:
Modals and snapshots don't get along with jest, so I think we can remove the `toMatchSnapshot()` calls in the tests for these. Snapshots here are empty and don't contain anything useful